images/kubevirt-infra-bootstrap/Dockerfile

```diff
@@ -16,3 +16,8 @@ RUN apt-get update \
         python3 \
         python3-pip \
         && rm -rf /var/lib/apt/lists/*
+RUN export KUSTOMIZE_DIR=/opt/kustomize \
+    && mkdir -p $KUSTOMIZE_DIR \
+    && cd $KUSTOMIZE_DIR \
+    && curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash \
```

Comment:
We usually try to avoid using master / latest. It introduces some risk when it updates to a new, maybe incompatible version, which will break CI. Do you mind using a specific version?

Reply:
(and with this `curl ... | bash` flow it introduces even bigger risks)

Reply:
sure!

Reply:
and looking at the script, it always downloads the latest release.... so, please download a verioned kustomize binary from the release page, see e.g. https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv3.5.4